src/OrchardCore.Commerce.Web/OrchardCore.Commerce.Web.csproj

```diff
@@ -3,9 +3,15 @@
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
     <CopyRefAssembliesToPublishDirectory>true</CopyRefAssembliesToPublishDirectory>
+
+    <NodeJsExtensionsMarkdownAnalysisMode>solution</NodeJsExtensionsMarkdownAnalysisMode>
+    <ExecDotnetPostcleanCommand>false</ExecDotnetPostcleanCommand>
+    <ExecDotnetPrebuildCommand>false</ExecDotnetPrebuildCommand>
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Lombiq.Analyzers.PowerShell" Version="1.0.0" />
+    <PackageReference Include="Lombiq.NodeJs.Extensions" Version="1.0.0-alpha.17" />
```

Comment:
Document what one needs to be installed locally to be able to build the solution.

Reply:
[Apparently nothing?](https://github.com/Lombiq/NodeJs-Extensions/blame/dev/Readme.md#L37)

I guess it helps if you have Node.js installed. :smile_cat: I've added that to the readme. I have just installed a new Linux setup yesterday (Márk suggested a new distro that's right up my alley) so I can confirm that only installing Node (via NVM) is enough.

Reply:
btw this isn't news, the project was [already included in a different module](https://github.com/OrchardCMS/OrchardCore.Commerce/pull/127/files#diff-d1a8d49b5b14a1cdaa27b65272d273762c30616abcfa89fe6f350946c5d9a2bbR14) as it was needed for the Stripe integration. We just didn't think to document it, (probably because pretty much everyone here already has it installed for other projects) so thanks for highlighting that blind spot!

Reply:
> a new distro that's right up my alley

@DAud-IcI Which one is it?

Reply:
[EndevourOS](https://endeavouros.com/)